tests/testing_harness.py

```diff
@@ -312,3 +312,15 @@ class HashedPyAPITestHarness(PyAPITestHarness):
     def _get_results(self):
         """Digest info in the statepoint and return as a string."""
         return super()._get_results(True)
+
+
+class CADTestHarness(TestHarness):
```

Comment:
I don't think we need a special test harness for this test. Do we?

Reply:
Once I add expected result files, no we won't.

Reply:
Gotcha. Are you planning on doing that as part of this PR?

Reply:
Definitely. In the process of switching machines, but should have that done soon.